tests/acceptance/features/apiSpaces/listSpaces.feature

```diff
@@ -423,6 +423,87 @@ Feature: List and create spaces
     Then the HTTP status code should be "404"
     And the json responded should not contain a space with name "Project Venus"
     Examples:
-      | role        |
+      | role       |
+      | User       |
+      | User Light |
+
+  @issue-7160
+  Scenario Outline: get share jail space information of the user when user has a pending share
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has disabled auto-accepting
+    And user "Brian" has uploaded file with content "this is a test file." to "test.txt"
+    And the administrator has assigned the role "<userRole>" to user "Alice" using the Graph API
+    When user "Brian" shares file "/test.txt" with user "Alice" using the sharing API
+    And user "Alice" lists all available spaces via the GraphApi
```

Comment:
can the steps be?
```suggestion
    Given user "Brian" has shared file "/test.txt" with user "Alice" using the sharing API
    When user "Alice" lists all available spaces via the GraphApi
```

Reply:
No according to https://github.com/owncloud/ocis/issues/7160#issuecomment-1699328421 the share needs to be on pending state and I think `has shared` would accept the share as well?

Reply:
>I think has shared would accept the share as well?

I don't think so :thinking: or does it? then it should not

Reply:
Not sure. If it doesn't accept then we can use that. I'll look

Reply:
I changed it to given step as we don't accept share.